spacy/vocab.pyx

```diff
@@ -157,7 +157,7 @@ cdef class Vocab:
                                                   orth=key, orth_id=string))
             return lex
         else:
-            return self._new_lexeme(mem, string)
+            return self._new_lexeme(string)
```

Comment:
Since this PR targets `v4`, I think it'd make sense to also propagate the change upward, i.e., remove the now-unused `mem` parameter from `Vocab.get` and update its callsites?